index.html

```diff
@@ -2968,26 +3000,47 @@ <h3>Data Schema Serialization</h3>
 </section> <!-- end of id="data-schema-serialization-json"-->
 
 <section id="titles-descriptions-serialization-json">
-<h3><code>title(s)</code> and <code>description(s)</code></h3>
+<h3>Human-Readable Metadata</h3>
 
   <p>
   The vocabulary terms <code>title</code> and <code>description</code> are 
   used to provide human-readable texts for titles (e.g., a display
   text for UI) and additional information useful for <a>Consumers</a> of TDs.
-  When a default language is specified by assigned a value to <code>@language</code>
+  When a default language is specified by assigning a value to <code>@language</code>
   in the <code>@context</code> of a <a href="#sec-thing-as-a-whole-json">Thing object</a>,
   that default language is associated with the string values assigned to
   <code>title</code> and <code>description</code> within the TD
-  and to <code>name</code> in the Thing object.
+  and to <code>name</code> and <code>support</code> in the Thing object.
+  </p>
+  <p>
   Another possibility is to set the default language with
-  the language negotiation mechanism of HTTP
-  (i.e., the <code>Accept-Language</code> header field).
+  a language negotiation mechanism, such as 
+  the <code>Accept-Language</code> header field of HTTP.
+  <span class="rfc2119-assertion" id="td-ns-multilanguage-content-negotiation">
+  In protocols such as HTTP that have an ability to negotiate content based on languages,
+  terms (e.g. <code>description</code> and <code>title</code>) intended
+  for carrying texts in the default language SHOULD be used instead of <code>MultiLanguage</code> container
+  when the texts are served as a result of such negotiation.</span>
```

Comment:
In protocols such as HTTP that have an ability to negotiate content based on languages,
TD documents SHOULD only include JSON members based on the default language
(e.g., <code>description</code> and <code>title</code>)
and no <code>MultiLanguage</code> objects,
while the <code>@language</code> member is omitted.

Reply:
I am fine with adding an assertion to leave out the MultiLanguage objects.   I'm not convinced it is a good idea to leave out @language since if the TD gets forwarded somewhere it would be useful.  I think instead we should add an assertion that if it is included, it MUST be consistent with the negotiated default language.

Reply:
I added the following two assertions; notice the first one mandates leaving OUT MultiLanguage objects which seems simpler.
* In contexts where the default language has been negotiated, TD documents SHOULD NOT include any MultiLanguage objects. 
* In contexts where the default language has been negotiated, a @language definition, if present, MUST be consistent with the negotiated default language.

Reply:
Good